src/git.js

```diff
@@ -141,7 +140,7 @@ class Git {
 
 	async add(file) {
 		return execCmd(
-			`git add -f ${ file }`,
+			`git add -f "${ file }"`,
```

Comment:
Added because I had problems when testing files with spaces. Maybe because of Windows, I don't know why it didn't failed to you :)

Reply:
Could be, haven't tried run the action on Windows